### ISSUE TITLE
Option for tasks to stack upwards, rather than shift the progress bars

### ISSUE THREAD

**Issue by tjdevries** [j-hui/fidget.nvim#1]
With the following diff (just a hack, worked OK for me) I was able to make it so that the lines don't shift around so much using the plugin. I really like the spinner, but found it distracting when the spinner line itself moves up and down a lot depending on how many tasks exist. This way, the  tasks get stacked _on top_ of the spinner, and the spinner stays in the same location the whole time.

(I can record video if this doesn't make sense)

```diff
❯ cat out.diff                           
diff --git a/lua/fidget.lua b/lua/fidget.lua
index cdd20a7..9029a9e 100644
--- a/lua/fidget.lua
+++ b/lua/fidget.lua
@@ -63,7 +63,7 @@ function base_fidget:fmt()
   self.max_line_len = #line
   for _, task in pairs(self.tasks) do
     line = options.fmt.task(task.title, task.message, task.percentage)
-    table.insert(self.lines, line)
+    table.insert(self.lines, 1, line)
     self.max_line_len = math.max(self.max_line_len, #line)
   end
   if options.fmt.leftpad then
@@ -114,7 +114,7 @@ function base_fidget:show(offset)
   api.nvim_win_set_option(self.winid, "winblend", 100) -- Make transparent
   api.nvim_win_set_option(self.winid, "winhighlight", "Normal:FidgetTask")
   api.nvim_buf_set_lines(self.bufid, 0, height, false, self.lines)
-  api.nvim_buf_add_highlight(self.bufid, -1, "FidgetTitle", 0, 0, -1)
+  api.nvim_buf_add_highlight(self.bufid, -1, "FidgetTitle", height - 1, 0, -1)
 
   return #self.lines + offset
 end
```

**Comment by j-hui** [comment]
Oh that's a good idea! I'll expose this as an option.

**Comment by j-hui** [comment]
Closed by #2

**Comment by tjdevries** [comment]
Thanks, I love it :)
